### PR TITLE
SRM RN: version 1.2.5

### PR DESCRIPTION
**Pull request by SudheendraKatte**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: release-notes/service-reliability-management.md [hunk 1]
```diff
@@ -18,11 +18,11 @@ Review the notes below for details about recent changes to Harness Service Relia
 Harness deploys changes to Harness SaaS clusters on a progressive basis. This means that the features and fixes that these release notes describe may not be immediately available in your cluster. To identify the cluster that hosts your account, go to the **Account Overview** page. 
 :::
 
-## Latest: Version 1.1.3
+## Latest: Version 1.2.5
 
 ### New features and enhancements
 
-SRM dashboards now include comprehensive data for composite Service Level Objectives (composite SLOs). This enhancement gives you a holistic view of both simple and composite SLO performance. (SRM-15419)
+Added new filters, Environment-Based and SLO Type, to the SLO listing page. You can now filter the SLO list based on environment or SLO type for improved management. (SRM-15506)
```
Comment (deepakbolangady) [inline release-notes/service-reliability-management.md:25]:
```suggestion
Added new filters named Environment-Based and SLO Type to the SLO listing page. You can now filter the SLO list on environment or SLO type for improved management. (SRM-15506)
```
(If you wish to use "based on", I would recommend "on the basis of" in this scenario, but even that's not necessary. Just "on" will do.)

File: release-notes/service-reliability-management.md [hunk 2]
```diff
@@ -31,10 +31,9 @@ This release does not include any early access features.
 
 ### Fixed issues
 
-- FireHydrant notifications were not being triggered as expected for monitored services without any configured Service Level Objectives (SLOs). (SRM-15458)
-
-  This issue has been resolved. FireHydrant notifications will now be triggered for all monitored services, irrespective of whether SLOs are configured.
+- Previously, when any of the simple Service Level Objectives (SLOs) reported NO_DATA for the entire duration, and if the missing data was treated as IGNORE, the composite SLO faced challenges as it necessitated data from all underlying simple SLOs to perform calculations. Consequently, this resulted in the composite SLO failing to produce any records and becoming unresponsive. (SRM-15376)
```
Comment (deepakbolangady) [inline release-notes/service-reliability-management.md:34]:
```suggestion
- Previously, when any of the simple service-level objectives (SLOs) reported NO_DATA for the entire duration, and the missing data was treated as IGNORE, the composite SLO faced challenges as it required data from all underlying simple SLOs to perform calculations. Consequently, the composite SLO failed to produce any records and became unresponsive. (SRM-15376)
```
> faced challenges 

This is very anthropomorphic. Can we write something else? 

> NO_DATA and IGNORE

Can we replace these strings with words that describe what they represent? 

Would the following construction be an improvement?

> Composite SLOs require data from all underlying simple SLOs to perform calculations. Therefore, previously, if any of the simple service-level objectives (SLOs) failed to report data for the entire duration, and that missing data was ignored, the composite SLO failed to produce any records and became unresponsive. (SRM-15376)

File: release-notes/service-reliability-management.md [hunk 3]
```diff
@@ -31,10 +31,9 @@ This release does not include any early access features.
...
 
+  This issue has been resolved. Now, when one or more simple SLOs report NO_DATA throughout the duration, even when the missing data is treated as IGNORE, the composite SLO continues to generate records with data for all minutes. Importantly, the error budget for those minutes remains unaffected.
```
Comment (deepakbolangady) [inline release-notes/service-reliability-management.md:36]:
```suggestion
  This issue has been resolved. Now, if one or more simple SLOs report NO_DATA throughout the duration, even if the missing data is treated as IGNORE, the composite SLO continues to generate records with data for the entire duration. Importantly, the error budget for those minutes remains unaffected.
```

**Review by deepakbolangady — APPROVED** [review]
Approved with some feedback.

**Comment by SudheendraKatte** [comment]
Created another PR -  [3575](https://github.com/harness/developer-hub/pull/3575) resolve conflicts. Closing this PR.

**Comment by SudheendraKatte** [comment]
Closing this PR due to multiple conflicts and broken links. Refer the new PR https://github.com/harness/developer-hub/pull/3575

**Pull request closed without merging**